src/app/containers/StoryPromo/index.jsx

```diff
@@ -12,6 +13,7 @@ import Timestamp from '../Timestamp';
 
 const StoryPromo = ({ item }) => {
   const headline = deepGet(['headlines', 'headline'], item);
+  const url = deepGet(['locators', 'assetUri'], item);
```

Comment:
Could we maybe return null in this file if `url` or `headline` are missing? I think thats kinda the bare minimum for a story promo to be useful. Thoughts?

Reply:
Mmm It makes sense. What about the image? Would be displaying a promo if we don't have it?

Reply:
I think no image is fine, its just the bare minimum to still serve a purpose